docs/modeling/index.rst

```diff
@@ -10,337 +10,121 @@ Introduction
+these points. Examples on using models can be found
+:ref:`here <modeling_tutorial>`.
 
-    `astropy.modeling` is currently a work-in-progress, and thus it is likely
-    there will still be API changes in later versions of Astropy.  Backwards
-    compatibility support between versions will still be maintained as much as
-    possible, but new features and enhancements are coming in future versions.
-    If you have specific ideas for how it might be improved, feel free to let
-    us know on the `astropy-dev mailing list`_ or at
-    http://feedback.astropy.org
+Detailed documentation on models can be found :doc:`here <models>`.
+Additional features of models are:
 
+Advanced Model Topics
+---------------------
 
-Getting started
-===============
+* **Compound models**
 
-The examples here use the predefined models and assume the following modules
-have been imported::
+  Compound models can be created by combining existing models with
+  arithmetic expressions. A quick tutorial of compound models can be found
+  :ref:`here <compound-models-intro>`. For more detailed documentation on
+  compound models, see :ref:`compound-models`.
 
-    >>> import numpy as np
-    >>> from astropy.modeling import models, fitting
+* **Model Sets**
 
+  Users can also create a model *set* which is a collection of models of the same
+  type, but with different values for their parameters. See
+  :ref:`here <model-sets-index>` for a quick tutorial on model sets.
 
-Using Models
-------------
+* **Custom models**
 
-The `astropy.modeling` package defines a number of models that are collected
-under a single namespace as ``astropy.modeling.models``.  Models behave like
-parametrized functions::
+  The package is intended to allow users to easily create their own models.
+  See documentation :ref:`here<modeling-new-classes>`.
 
-    >>> from astropy.modeling import models
-    >>> g = models.Gaussian1D(amplitude=1.2, mean=0.9, stddev=0.5)
-    >>> print(g)
-    Model: Gaussian1D
-    Inputs: ('x',)
-    Outputs: ('y',)
-    Model set size: 1
-    Parameters:
-        amplitude mean stddev
-        --------- ---- ------
-              1.2  0.9    0.5
+* **Algorithms**
 
-Model parameters can be accessed as attributes::
+  A description of the algorithms used in the 1-D polynomial, Chebyshe
+  and Legendre polynomial models can be found :doc:`here <algorithms>`.
 
-    >>> g.amplitude
-    Parameter('amplitude', value=1.2)
-    >>> g.mean
-    Parameter('mean', value=0.9)
-    >>> g.stddev
-    Parameter('stddev', value=0.5)
+Parameters
+----------
 
-and can also be updated via those attributes::
+This refers to the parameters of a model which are implemented as attributes of
+models. For example, the parameters of a ``Gaussian1D`` model are
+``amplitude``, ``x_0`` and ``stddev`` which refer to the amplitude, the
+position of the center of the peak, and the standard deviation of the function
+respectively. Features of parameters are:
```

Comment:
Instead of "are" I might say "include" since this list is not exhaustive.  Might also be worth pointing out that parameters can be arrays.